CMakeLists.txt

```diff
@@ -1,3 +1,26 @@
+## CMake configuration file of glog project
+##
+## When this project is a subproject. The "glog::glog" target is in this case
+## an ALIAS library target for the "glog" library target. Targets which depend
+## on the glog library should link to the "glog::glog" library target.
+##
+## Example CMakeLists.txt of user project which requires separate glog installation:
+##   cmake_minimum_required(VERSION 2.8.12 FATAL_ERROR)
+##   project(Foo)
+##
+##   find_package(glog REQUIRED)
+##
+##   add_executable(foo src/foo.cc)
+##   target_link_libraries(foo glog::glog)
+##
+## Example CMakeLists.txt of super-project which contains glog source tree:
+##   cmake_minimum_required(VERSION 2.8.12 FATAL_ERROR)
+##   project(Foo)
+##
+##   add_subdirectory(glog)
+##
+##   add_executable(foo src/foo.cc)
+##   target_link_libraries(foo glog::glog)
```

Comment:
I believe, this is not a good place to document such specific stuff. I would suggest moving this part either into the wiki or `cmake/INSTALL.md`.

Reply:
Ok, I'll move it...

ps: I just put it here as [gflags CMakelists.txt](https://github.com/gflags/gflags/blob/master/CMakeLists.txt#L21) did